tempesta_fw/t/functional/run_tests.py

```diff
@@ -154,6 +154,17 @@ def usage():
 
 
 remote.connect()
+
+# configuring TCP
```

Comment:
Since you're modifying tcp sysctls, it's obvious that you configure TCP. It's better to write a more programmer-friendly comment explaining that you make TCP sockets recycle quicker to make next tests run using the same TCP ports.

Reply:
fixed